Pod/Classes/CDTQQuerySqlTranslator.m

```diff
@@ -350,22 +353,31 @@ + (CDTQSqlParts *)wherePartsForAndClause:(NSArray *)clause usingIndex:(NSString
                     [sqlParameters addObject:[negatedPredicate objectForKey:operator]];
 
             } else {
+                NSString *sqlClause;
                 NSString *sqlOperator = operatorMap[operator];
                 NSString *tableName = [CDTQIndexManager tableNameForIndex:indexName];
-                NSString *sqlClause = [CDTQQuerySqlTranslator whereClauseForNot:fieldName
-                                                                  usingOperator:sqlOperator
-                                                                       forTable:tableName];
-                
-                [sqlClauses addObject:sqlClause];
-                predicateValue = [negatedPredicate objectForKey:operator];
-                if ([self validatePredicateValue:predicateValue]) {
-                    [sqlParameters addObject:predicateValue];
+                NSString *placeholder;
+                if ([operator isEqualToString:IN]) {
+                    // The predicate dictionary value must be an NSArray here.
+                    // This was validated during normalization.
+                    NSArray *inList = [negatedPredicate objectForKey:operator];
```

Comment:
Now using shorthand NSDictionary access to values throughout Translator in b667c92d0cf9994ca7a8751af6e7efedcb6f3199.